_includes/releases/v22.1/v22.1.0-alpha.4.md

```diff
@@ -0,0 +1,97 @@
+## v22.1.0-alpha.4
+
+Release Date: March 21, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-alpha-4-security-updates">Security updates</h3>
+
+- Clusters can be configured to send HSTS headers with HTTP requests in order to enable browser-level enforcement of HTTPS for the cluster host. This is controlled by setting the `server.hsts.enabled` [cluster setting](../v22.1/cluster-settings.html) to `true` (default: `false`). Once the headers are present, after an initial request, browsers will force HTTPS on all subsequent connections to the host. This reduces the possibility of man-in-the-middle (MitM) attacks, which HTTP-to-HTTPS redirects are vulnerable to. [#77244][#77244]
+
+<h3 id="v22-1-0-alpha-4-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- Added a `created` time column to the `crdb_internal.active_range_feeds` virtual table to improve observability and debuggability of the rangefeed system. [#77597][#77597]
+- Incremental backups created by [`BACKUP ... INTO`](../v22.1/backup.html) or [`BACKUP ... TO`](../v22.1/backup.html) are now stored by default under the path `incrementals` within the backup destination, rather than under each backup's path. This enables easier management of cloud-storage provider policies specifically applied to [incremental backups](../v22.1/take-full-and-incremental-backups.html#incremental-backups). [#75970][#75970]
+
+<h3 id="v22-1-0-alpha-4-sql-language-changes">SQL language changes</h3>
+
+- Added a `sql.auth.resolve_membership_single_scan.enabled` [cluster setting](../v22.1/cluster-settings.html), which changes the query for an internal role membership cache. Previously the code would recursively look up each role in the membership hierarchy, leading to multiple queries. With the setting on, it uses a single query. The setting is `true` by default. [#77359][#77359]
+- The [data type](../v22.1/data-types.html) of shard columns created for [hash-sharded indexes](../v22.1/hash-sharded-indexes.html) has changed from `INT4` to `INT8`. This should have no effect on behavior or performance. [#76930][#76930]
+- Introduced the `sql.contention.resolver.queue_size` metric. This gauge metric gives the current length of the queue of contention events, each awaiting translation of its transaction ID into a transaction fingerprint ID. This metric can be used to assess the level of backlog unresolved contention events. [#77514][#77514]
```

Comment:
It looks like the next two bullet points are missing an article in front of "transaction fingerprint ID". Seems like the first instance used "a transaction fingerprint ID"